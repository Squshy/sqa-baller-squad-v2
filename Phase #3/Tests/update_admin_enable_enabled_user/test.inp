LOGIN
ADMINBOY
AdPass123
UPDATE
HECKING COOLMAN 
ENABLE