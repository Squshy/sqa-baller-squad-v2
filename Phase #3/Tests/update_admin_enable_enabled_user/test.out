=========================================================
                        UPDATE
=========================================================
Would you like to enable or disable a user?: DISABLE

Which user would you like to enable?: HECKING COOLMAN
---------------------------------------------------------
HECKING COOLMAN's account is already enabled.
=========================================================